src/index.html

```diff
@@ -244,17 +244,21 @@ <h2 class="gallery__title">Галерея</h2>
             <h2 class="follow__title">Підписка</h2>
             <p class="follow__text">Бути в курсі всіх актуальних подій та останніх новин.</p>
             <div class="follow__button">
-              <form class="follow__form">
+              <form
+                action="mailto:info@namu.kiev.ua"
```

Comment:
[checklist](https://github.com/mate-academy/Museum/blob/master/checklist.md) point 9
try to clear the form and scroll to the top instead of sending an email

Reply:
I hope I understood correctly.